server/resources/META-INF/cloudstack/core/spring-server-core-managers-context.xml

```diff
@@ -271,6 +271,6 @@
         class="org.apache.cloudstack.region.gslb.GlobalLoadBalancingRulesServiceImpl" >
         <property name="gslbServiceProviders" value="#{gslbServiceProvidersRegistry.registered}" />
     </bean>
-    <bean id="certServiceImpl" class="org.apache.cloudstack.network.lb.CertServiceImpl" />
+    <bean id="certServiceImpl" class="org.apache.cloudstack.network.ssl.CertServiceImpl" />
```

Comment:
@rhtyd we will address that when we make this into a CA-service. I'll create a ticket. It is really very minor.